### DISCUSSION TITLE
Openwrt下怎么在线更新呢？

### DISCUSSION THREAD

**Discussion by yuguod** [messense/aliyundrive-webdav#535] [Q&A]
这个软件因为都知道的原因更新比较频繁，但是OpenWrt用户怎么才能更新呢？毕竟不能每次更新都重新编译固件，能不能开发在线更新功能呢？

**Comment by master-yun-yun** [comment]
1.6.0版本就增加自动更新功能了
不过openwrt下确实没有自动更新
我都是每次卸载，再重装新版本
